### ISSUE TITLE
Express semantics of Glk styles for screen readers

### ISSUE THREAD

**Issue by curiousdannii** [curiousdannii/asyncglk#20]
The Glk styles have inherent semantics, but that is completely lost in WebGlkOte, and any visual representation of those styles is not accessible to those using screen readers. But it might be possible to wrap the current output of `<span>`s with HTML elements that express the meaning of the Glk styles. We would have to carefully ensure that any default CSS properties for these elements are reset, as well as making them all inline. So for example, here is how the subheader style, used for a room description, could be output:

```
<h3><span class="Style_subheader">Kitchen Garden</span></h3>
```

Current interpreters like ZVM misuse some of these Glk styles for purely visual formatting. But we could add a Glk extension to allow for broader CSS properties to be set so that ZVM can just say "make this text bold and italic", and then "headers" and "blockquotes" etc wouldn't be used in weird situations.

Possible ways of expressing the Glk semantics:

| Glk style | HTML option |
|----------|----------------|
| Normal | none |
| Emphasized | `<em>` |
| Preformatted | `<pre>`? |
| Header | `<h2>` |
| Subheader | `<h3>` |
| Alert | `role="alert"` |
| Note | ? |
| Blockquote | `<blockquote>` |
| Input | ? |
| User1 | none |
| User2 | none |
